src/main/java/com/notably/model/suggestion/SuggestionModel.java

```diff
@@ -29,13 +29,13 @@
      * TODO: add logo.
      * @return The meaning of the user input.
      */
-    Property<Optional<String>> commandTextProperty();
+    Property<Optional<String>> getCommandTextProperty();
 
     /**
      * Sets the command text.
-     * @param commandText The "command" that the user inputs, e.g. open, search, etc.
+     * @param commandInputText The "command" that the user inputs, e.g. open, search, etc.
      */
-    void setCommandInputText(String commandText);
+    void setCommandInputText(String commandInputText);
 
     String getCommandInputText();
```

Comment:
Sorry I missed these in the previous PRs.

Actually, why do you need these methods? You can access these from the `Model` class.
Instead, you should have a `setCommandText` method that updates the `commandTextProperty`

Perhaps you'd wanna come up with a name that better differentiates `commandInputText` and `commandText` too. Maybe we rename `commandText` to `suggestionResponseText` or just `responseText`?